webapp/src/components/tutorial/tours.ts

```diff
@@ -24,6 +25,12 @@ export const PB_TOUR_EX = {
     FINISHED,
 };
 
+export const PlaybookEditTutorialSteps = {
+    Checklists: 0,
+    Actions: 1,
+    StatusUpdates: 2,
+    Retrospective: 3,
```

Comment:
Just needs a `FINISHED` to get the 4th dot to show. Not sure why this isn't just built in if it's expected 🤷‍♂️!

```suggestion
    Retrospective: 3,
    FINISHED,
```
<img width="300" alt="CleanShot 2022-02-09 at 21 43 46@2x" src="https://user-images.githubusercontent.com/11724372/153333584-8006e0cf-46ca-4574-814e-a89b63fd6b40.png">

<img width="300" alt="CleanShot 2022-02-09 at 21 42 43@2x" src="https://user-images.githubusercontent.com/11724372/153333418-34fa27db-b1c4-4aa5-a93a-20dfe6cf978f.png">